src/components/SideNav/MobileMenu/index.jsx

```diff
@@ -62,7 +63,20 @@ export const MobileMenu = ({ onClose, ...rest }) => {
             undefined
           }
         >
-          <img style={{ marginRight: '.75rem' }} width={'20px'} src={farming} alt="farming" />
+          <Icon
+            icon={
+              <Farms
+                height={20}
+                width={20}
+                color={
+                  history.location.pathname.split('/')[1] === 'farming' ||
+                  history.location.pathname.split('/')[1] === 'farming'
```

Comment:
🤔

Reply:
Didn't notice that, we can remove it :+1: